Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
        p = FileProcessor("filename.py", options_from(), lines=["import sys\n"] * 10)
        with p.inside_multiline(2):
            pass
>       assert not p.multiline
E       assert not True
E        +  where True = <flake8.processor.FileProcessor object at 0x77f31e7dc700>.multiline

../../test.py:22: AssertionError
=========================== short test summary info ============================
FAILED ../../test.py::test - assert not True
============================== 1 failed in 0.04s ===============================
